app/lib/actions/katello/event_queue/poller_thread.rb

```diff
@@ -26,15 +26,17 @@ def poll_for_events(suspended_action)
           @thread.kill if @thread
           @thread = Thread.new do
             loop do
-              begin
-                until (event = ::Katello::EventQueue.next_event).nil?
-                  suspended_action.notify_queue_item(event.event_type, event.object_id, event.created_at) if event
-                end
+              Rails.application.reloader.wrap do
+                begin
+                  until (event = ::Katello::EventQueue.next_event).nil?
+                    suspended_action.notify_queue_item(event.event_type, event.object_id, event.created_at) if event
+                  end
 
-                sleep SLEEP_INTERVAL
-              rescue => e
-                suspended_action.notify_fatal(e)
-                raise e
+                  sleep SLEEP_INTERVAL
+                rescue => e
```

Comment:
I didn't add this line (just indented it), should this be a rubocop execption or do we want to refactor it?

Reply:
You can just change it to:

```
rescue Exception => e
```

which should do the exact same thing but also quiet rubocop. I'd say go ahead and change it, since we kind of have the policy of 'touch it, fix it' ;)